lib/manageiq/loggers/journald.rb

```diff
@@ -28,12 +44,59 @@ def add(severity, message = nil, progname = nil)
         end
 
         message = formatter.call(format_severity(severity), progname, message)
+        caller_object = caller_locations.last
 
-        Systemd::Journal.print(log_level_map[severity], message)
+        # TODO: Allow the syslog facility to be set via the configuration settings.
+        Systemd::Journal.message(
+          :message           => message,
+          :message_id        => message_id,
+          :priority          => log_level_map[severity],
+          :application       => 'manageiq',
```

Comment:
We might want the application to be configurable also, at a minimum for productization

Reply:
Ideally, yes, though it has to be hardcoded for now, since `Settings` aren't available when this is loaded. Or is there a way?

Reply:
We can make it an accessor and default to `manageiq` in `initialize`

Reply:
Done!